webhook_server.py

```
@@ -595,6 +640,300 @@ async def get_blog_assets(date: str):
+    if not signature or not timestamp:
+        return False
+    
+    try:
+        # Create the message to verify
+        message = timestamp.encode() + body
+        
+        # Verify the signature
+        public_key_bytes = bytes.fromhex(DISCORD_PUBLIC_KEY)
+        verify_key = nacl.signing.VerifyKey(public_key_bytes)
+        verify_key.verify(message, bytes.fromhex(signature))
+        return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)